boards/arm64/roc_rk3568_pc/roc_rk3568_pc.dts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2022 HNU
+ * Copyright 2022 openEuler SIG-Zephyr
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <arm64/rockchip/rk3568.dtsi>
+#include <common/mem.h>
+
+/ {
+	model = "Firefly ROC-RK3568-PC";
+	compatible = "rockchip,rk3568";
+
+	chosen {
+		zephyr,console = &uart2;
+		zephyr,shell-uart = &uart2;
+		zephyr,sram = &sram0;
+	};
+
+	sram0: memory@40000000 {
+		compatible = "memory";
```

Comment:
This compatible doesn't exist. Do you mean `mmio-sram`?

Reply:
This SoC's sram is just 64K, and it too small for some test cases. So I  put the zephyr image to the ddr. 
By the way, the compatible seem appers below:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/40254934/3e255482-fde6-4afd-ade1-716a0cf94126)
I saw there is `memory` on above, so I also added it.

Reply:
that is wrong. If that is SRAM the `mmio-sram` compatible must be present.

Reply:
OK, I understand it.